css-canvas-medical/index.html

```diff
@@ -0,0 +1,108 @@
+        <img class="logo" src="images/logo-blue.svg">
+      </div>
+      <div class="column-three-fifths padding-left-40">
+        <a href="#" class="font-avenir color-light-grey font-size-22 padding-right-60">Product</a>
+        <a href="#" class="font-avenir color-light-grey font-size-2 padding-right-60">Implementation</a>
+        <a href="#" class="font-avenir color-light-grey font-size-22 padding-right-60">About Us</a>
+        <a href="#" class="font-avenir color-light-grey font-size-22 padding-right-60">Blog</a>
+        <button class="text-white blue-button-small font-avenir font-size-22">Contact Us</button>
+      </div>
+    </div>
+    <hr>
+    <div class="row">
+      <div class="column-full hero text-white">
+        <div class="overlay"></div>
+        <h1 class="font-tiempos hero-title">The EMR Reimagined for <br> Primary Care</h1>
+        <p class="font-avenir font-size-22 hero-text">Can't remember the last time you talked with a patient without
+          focusing on the EMR? Canvas lets you rediscover joy in
+          the work you do. Capture meaningful data, identify gaps in care, and move seamlessly between tasks within the
+          platform. You stay present with patients while your whole care team stays up-to-speed. Be a better doctor with
+          Canvas,
+          designed for primary care providers with the help of primary care providers.</p>
+        <button class="text-white blue-button font-avenir margin-top-30 hero-button">Get Started</button>
+      </div>
+    </div>
+    <div class="row background-off-white color-grey display-block padding-bottom-50 padding-top-80">
+      <div class="column-full text-align-center">
+        <h2 class="font-tiempos margin-5">Clinicians Prefer Canvas</h2>
+        <p class="font-avenir font-size-22 line-height">Physician-owned primary care practices across the country have
+          <br>already made Canvas the fabric of their care.</p>
+      </div>
+    </div>
+    <div class="row background-off-white color-grey font-size-24 font-avenir padding-bottom-100">
+      <div class="column-one-half text-align-center padding-topbottom-20 padding-left-80 font-size-27 line-height">
+        <p>Cascade Family Practice, Portland OR</p>
+        <p>Temescal Creek Medicine, Oakland CA</p>
+        <p>San Juan Healthcare, Friday Harbor WA</p>
+      </div>
+      <div class="column-one-half text-align-center padding-topbottom-20 font-size-27 line-height">
+        <p>Ward Medical Clinic, Bonifay FL</p>
+        <p>Caribou Medical, Towanda PA</p>
+        <p>Bay Tree Family Care, Red Bay AL</p>
+      </div>
+      <hr>
+    </div>
+    <div class="row">
+      <div class="column-one-third color-grey text-align-left padding-left padding-bottom-30">
+        <h2 class="font-tiempos">Ready to learn more?</h2>
+        <p class="font-avenir font-size-22">Schedule a private meeting now</p>
+      </div>
+      <div class="column-one-third">
+      </div>
```

Comment:
Uh oh, this is bad news. You should never use empty columns to create space. Instead, you can make the previous element `column-two-thirds` or make use of a utility class with `justify-content` to control where the items in this row are aligned.